newdle/client/src/locales/en/messages.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable */ module.exports = {
```

Comment:
Is this a generated file? If yes, we should probably not include it in the repo and gitignore it.

Reply:
maybe check if we can automatically build it during `make react-server` (could just be done in the makefile before launching the actual react server - i don't think we need auto rebuilding there as long as the english locale works even if strings are missing in here) and of course also in `make build`

Reply:
yes! I will add it to the gitignore then